### DISCUSSION TITLE
How to enable video upload on Media Page?

### DISCUSSION THREAD

**Discussion by abanghendri** [awcodes/filament-curator#509] [Q&A] [answered]
I install this plugin and use CuratorPlugin on my admin panel provider, when I visit media page and trye to upload video but the expected format only jpg, jpeg, png, svg and pdf, how to customize it and retriv all type of media including video and audio?

**Comment by awcodes** [comment]
What have you tried, code wise?

**Reply by abanghendri** [reply]
Nothing, just fresh install laravel, filament then curator, then adding CuratorPlugin to AdminPanelProvider, after that I can see media in my navigation sidebar, I open that and try to create/upload a media, but I cannot upload video/audio, it only expect images (jpg, jpeg, pdf, svg) by default

I try to dive into CuratorPlugin for some function posibilities but I cannot find how to add or extend posibiliyies to add video on my Media page

**Reply by awcodes** [reply, marked as answer]
publish the config and update the accepted file types.
